scripts/bootstrap/bootstrap

```diff
@@ -0,0 +1,336 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+#
+# bootstrap
+#
+# Install and configure Opkg, Entware and Toltec
+#
+# Arguments: Additional packages to install after setting up Toltec
+# Example: ./bootstrap harmony draft
```

Comment:
i was hoping it would be a line showing how to use it alongside curl or wget, f.e.: `wget -Qo- <url> | bash -s harmony draft`

Reply:
I understand, but I don’t feel comfortable recommending that users run a potentially unsafe chain of commands. And if we make it safe (i.e. include the file’s checksum in the example), then it’s no longer possible to include that line in the file itself.